quantecon/game_theory/brd.py

```diff
@@ -164,8 +164,26 @@ def __init__(self, payoff_matrix, N, epsilon=0.1):
 
     def play(self, action, action_dist, **options):
         """
-        See `play` in `BRD`.
+        Return a new action distribution using mutational best response.
+        
+        With probability epsilon, the player chooses a random action 
+        (mutation), otherwise plays the best response.
+        
+        Parameters
+        ----------
+        action : scalar(int)
+            Pure action of player who takes action.
 
+        action_dist : ndarray(int)
+            The initial action distribution of players.
+
+        **options : 
+            Keyword arguments passed to the best response method.
+            
+        Returns
+        -------
+        ndarray(int)
+            New action distribution.
```

Comment:
Again, maintaining the same docstring in two places (here and in `BRD.play`) increases the costs. I would keep it just as "See `play` in `BRD`.".